content/financial-aid.html

```diff
@@ -0,0 +1,81 @@
+    a community. It provides delegates with an opportunity to spend time with
+    their peers in a setting that allows the sharing of ideas, and exposes
+    people to opportunities and challenges that they may not otherwise get to
+    see. And above all else, it offers an opportunity to relax with and enjoy
+    the company of the wonderful people in our community!
+</p>
+
+<p>
+    Unfortunately, many members of our community are unable to take advantage
+    of this kind of opportunity for financial reasons. Attending a conference
+    is expensive, both in terms of explicit financial costs like ticket prices
+    and travel, but in terms of implicit costs like loss of time or the need
+    to find childcare.
+</p>
+
+<p>
+    These limitations mean that our community is unevenly exposed to the
+    benefits of conference attendance. This limits us: as long as only a subset
+    of our community are able to attend, we will be missing important
+    perspectives that make our events and lives richer.
+</p>
+
+<p>
+    For this reason, PyCon UK believes it is extremely important to enable as
+    many people as possible to attend. To this end, we are offering extensive
+    assistance in the form of financial aid. The goal of this program is to
+    enable people to attend who would like to, but cannot bear the financial
+    burden of attending.
+</p>
+
+<p>
+    Going hand-in-hand with the financial aid process is the other core tenet
+    of PyCon UK: <em>everybody pays</em>. This applies truly to everyone: by
+    default, PyCon UK asks that everybody buys a ticket, including the PyCon UK
+    Committee Chair, all speakers, and all volunteers. This system allows us to
+    support our substantial financial aid process, and ensures that discounted
+    admission is not granted to those who do not need it.
+</p>
+
+<p>
+    This means that it is <strong>extremely important</strong> that attendees
+    are proactive about applying for financial aid. If the only reason you are
+    not attending is that you cannot afford to do so, then
+    <strong>please</strong> apply for financial aid. You can find instructions
+    for doing so below.
+</p>
+
+<h2>Applying</h2>
+
+<p>
+    This year we are running our financial aid process alongside our ticket
+    sales, using the same service. This allows us to centralise our data
+    collection.
+</p>
+
+<p>
+    Please note that you <strong>do not</strong> need to have purchased a
+    ticket before applying for financial aid. On our financial aid form you'll
+    be able to ask that a ticket be set aside for you, conditional on your
+    grant being processed.
+</p>
+
+<p>
+    To apply for financial aid, visit <a href="about:blank">our Ti.To page</a>
+    and fill out the forms requested. We'll be in contact when we've decided
+    the status of your grant application.
+</p>
+
+<p>
+    The deadline for financial aid applications is (when?)
```

Comment:
From a purely financial perspective, I'm on board with that, because I've budgeted conservatively, and personally would aim to route as much of any Bonus Extra Money we get towards FinAid.

I imagine there are good logistical / marketing / cat-herding reasons to impose an artificial deadline though